README.md

```diff
@@ -120,6 +120,18 @@ To start our app, we need to generate both a rails-api app and an ember app insi
   ```no-highlight
   bundle install
   rails g rspec:install
+  ```
+
+* Add `shoulda-matchers` to your `spec/rails_helpler.rb` file.
+
+  ```ruby
+  require 'rspec/rails'
+  require 'shoulda/matchers'
```

Comment:
Do you really need this in your `rails_helper`? It's in the docs so I'll for sure add it but I have an app using `shoulda-matchers` without it. Weird.